tests/integration_tests/style/test_pylint.py

```diff
@@ -20,7 +20,8 @@ def test_python_pylint():
         '--variable-rgx="[a-z_][a-z0-9_]{1,30}$" --disable='
         "fixme,too-many-instance-attributes,import-error,"
         "too-many-locals,too-many-arguments,consider-using-f-string,"
-        "consider-using-with,implicit-str-concat,line-too-long"
+        "consider-using-with,implicit-str-concat,line-too-long,"
+        "broad-exception-raised"
```

Comment:
I'm personally fine with ignoring this one complain from pylint. It is not very useful. But maybe other have diff opinions.